appveyor.yml

```diff
@@ -8,7 +8,14 @@ environment:
 
 # Install scripts. (runs after repo cloning)
 install:
-  - ps: if (!(Test-Path $env:NPM_TOKEN)) { Rename-Item -Path .\.npmrc -NameName "npmrcdisabled"; }
+  - ps: |
+      if ($env:NPM_TOKEN.length -eq 0) {
```

Comment:
I'm not sure that I fully understand the original intent here. Should this check for the existence of a file with the same name as `NPM_TOKEN`?
If so, then maybe this should be changed to the following:
```ps
if (($env:NPM_TOKEN.length -gt 0) -and (!(Test-Path $env:NPM_TOKEN))) {
```
The problem with `Test-Path` alone is that it throws an error when its argument is null.